requirements.min

```diff
@@ -16,4 +16,5 @@ requests>=2.31.0
 rtree>=1.1.0
 scipy>=1.11.3
 shapely>=2.0.2
+openpyxl>=3.1.2
```

Comment:
what's openyxl needed for? I thought you are not parsing excel sheet anymore?

Reply:
ahh yes, I forgot. I'll remove this.

Reply:
Addressed in SLC PR